man/mblaze.7

```diff
@@ -3,81 +3,81 @@
 .Os
 .Sh NAME
 .Nm mblaze
-.Nd introduction to mblaze
+.Nd introduction to the mblaze message system
 .Sh DESCRIPTION
 The
 .Nm
-message system is a set of Unix utilities to deal with
-mail kept in Maildir folders.
+message system is a set of Unix utilities for processing and
+interacting with mail messages which are stored in maildir folders.
 .Pp
 Its design is roughly inspired by MH, the RAND Message Handling
 System, but it is a complete implementation from scratch.
 .Pp
 .Nm
-consists of these Unix tools that each do one job:
+consists of these Unix utilities that each do one job:
+.Pp
 .Bl -tag -width 11n -compact
 .It Xr maddr 1
-extract addresses from mail
+extract mail addresses from messages
 .It Xr magrep 1
-find mails matching a pattern
+print messages matching a pattern
```

Comment:
Does it really print the messages? (mshow does that...)

Perhaps `search messages matching a pattern` ala grep(1p).

Reply:
Yes, good. Done.